lms/djangoapps/affiliates/views.py

```diff
@@ -10,18 +10,27 @@
 from lms.djangoapps.ccx.models import CustomCourseForEdX
 from .models import AffiliateEntity, AffiliateMembership
 from django.contrib.auth.models import User
+from student.models import CourseEnrollment
 from lms.djangoapps.instructor.views.tools import get_student_from_identifier
 from .decorators import only_program_director, only_staff
 
 @only_staff
 def admin(request):
     affiliates = AffiliateEntity.objects.all().order_by('name')
     ccxs = CustomCourseForEdX.objects.all().order_by('display_name')
+    fasttrac_course_key = settings.FASTTRAC_COURSE_KEY
+
+    total_learners = CourseEnrollment.objects.filter(is_active=True).count()
+    total_fasttrac_learners = CourseEnrollment.objects.filter(is_active=True, course_id__startswith=fasttrac_course_key).count()
+    total_affiliate_learners = CourseEnrollment.objects.filter(is_active=True).exclude(course_id__startswith=fasttrac_course_key).count()
```

Comment:
Shouldn't you be able to just subtract total_fasttrac_learners from total_learners, judging by the queries?

Reply:
It should be easier to notice an query issue when written this way. If the total equals the sum of total A and total B, we are good.